db/migrate/20220125232709_create_verification_attempts.rb

```diff
@@ -0,0 +1,12 @@
+class CreateVerificationAttempts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :verification_attempts do |t|
+
```

Comment:
your note is exactly right -- just make sure you move lines 11 and 12 inside of the create_table block, and when you run `rails db:migrate`, it will add those references to the schema for the verification_attempts table.

(if you've already run `rails db:migrate`, you can run `rails db:rollback` to reverse the migration and then `rails db:migrate again once you've added the new fields)